client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-status-content.tsx

```diff
@@ -41,7 +43,7 @@ export default function SiteStatusContent( {
 	} = getRowMetaData( rows, type, isLargeScreen );
 
 	const siteId = rows.site.value.blog_id;
-	const siteUrl = value?.url;
+	const siteUrl = rows.site.value.url;
```

Comment:
Both provide the same value. `value.url` will be empty for types other than "site"